src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -243,6 +243,21 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
                             </ng-template>
                         </ngx-datatable-column>
                     }
+                    <ngx-datatable-column prop="" [sortable]="true" [minWidth]="100" [width]="100">
+                        <ng-template ngx-datatable-header-template>
+                            <span class="datatable-header-cell-wrapper d-inline-block w-100 text-center" (click)="controls.onSort('results?.last()?.assessmentNote?.note')">
+                                <fa-icon [icon]="faComment" [ngbTooltip]="'artemisApp.assessment.assessmentNote' | artemisTranslate" />
+                                <fa-icon [icon]="controls.iconForSortPropField('results?.last()?.assessmentNote?.note')" />
+                            </span>
+                        </ng-template>
+                        <ng-template ngx-datatable-cell-template let-value="value">
+                            @if (value.results?.last()?.assessmentNote?.note) {
+                                <span class="w-100 text-center">
+                                    <fa-icon [ngbPopover]="(value.results?.last()?.assessmentNote)!.note" [triggers]="'hover'" [icon]="faComment" />
+                                </span>
+                            }
+                        </ng-template>
```

Comment:
With your approach you would show the icon even if there is a complete empty text in the `assessmentNote.note`. That would lead to an icon without any popup, which is no desired.

Reply:
With `empty`, I do not have a string with only spaces in mind, but your idea is good. Applied with 2de1f59f3ebbb0caf859af6a86041e8b2d69da24.